project.clj

```diff
@@ -119,6 +119,9 @@
   ;; code that we have.
   :classifiers  [["test" :testutils]]
 
+  ;; Inovke "lein test :all" to run all tests
+  :test-selectors {:default (fn [m] (not (:slow m)))}
```

Comment:
Can we reflect this now in all the test scripts in ext ? Since I presume we want to do the :all case in CI.